src/Splunk.OpenTelemetry.AutoInstrumentation.Plugin/AssemblyInfo.cs

```diff
@@ -0,0 +1,19 @@
+﻿// <copyright file="AssemblyInfo.cs" company="Splunk Inc.">
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Splunk.OpenTelemetry.AutoInstrumentation.Plugin.Tests")]
```

Comment:
I think that you can remove whole file https://github.com/signalfx/splunk-otel-dotnet/blob/0006c29c5f6a99ef4e26c2ca2656a89325f3fa4a/src/Directory.Build.props#L36-L38